ci-operator/step-registry/rosa/setup/readiness/nodes/rosa-setup-readiness-nodes-commands.sh

```diff
@@ -240,8 +240,8 @@ else
 fi
 
 # Check if this is a HCP cluster
-is_hcp_cluster="$(rosa describe cluster -c "$CLUSTER_ID" -o json  | jq -r ".hypershift.enabled")"
-log "hypershift.enabled is set to $is_hcp_cluster"
+HOSTED_CP="$(rosa describe cluster -c "$CLUSTER_ID" -o json  | jq -r ".hypershift.enabled")"
```

Comment:
Be notice if this is already defined as env variable. Especially for old day1

Reply:
is_hcp_cluster is only used in the script. HOSTED_CP is defined as env variable and the default value is false.So I think it will not affect other jobs.Am I right?

Reply:
sounds low risk. I am OK with that.